bin/ssl.sh

```diff
@@ -45,7 +45,7 @@ if [ -e $CERTIFICATE ]; then
 fi
 
 # generates an RSA key
-openssl genrsa -out $SSLKEY 1024
```

Comment:
This change wasn't necessary but didn't see why we needed to be forcing a smaller size than the default.